plugins/Marketplace/vue/src/LicenseKey/LicenseKey.vue

```diff
@@ -0,0 +1,244 @@
+<!--
+  Matomo - free/libre analytics platform
+  @link https://matomo.org
+  @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+-->
+
+<template>
+  <div class="marketplace-max-width">
+    <div class="marketplace-paid-intro">
+      <div v-if="isValidConsumer">
+        <div v-if="isSuperUser">
+          {{ translate('Marketplace_PaidPluginsWithLicenseKeyIntro', '') }}
+          <br />
+          <div class="licenseToolbar valign-wrapper">
+            <DefaultLicenseKeyFields
+              :model-value="licenseKey"
+              @update:model-value="licenseKey = $event; updatedLicenseKey()"
+              @confirm="updateLicense()"
+              :has-license-key="hasLicenseKey"
+              :is-valid-consumer="isValidConsumer"
+              :enable-update="enableUpdate"
+            />
+            <SaveButton
+              class="valign"
+              id="remove_license_key"
+              @confirm="removeLicense()"
+              :value="translate('Marketplace_RemoveLicenseKey')"
+            />
+            <a
+              class="btn valign"
+              :href="subscriptionOverviewLink"
+            >
+              {{ translate('Marketplace_ViewSubscriptions') }}
+            </a>
+            <div v-if="showInstallAllPaidPlugins">
+              <a
+                href="javascript:;"
+                class="btn installAllPaidPlugins valign"
+              >
+                {{ translate('Marketplace_InstallPurchasedPlugins') }}
+              </a>
+              <div
+                class="ui-confirm"
+                id="installAllPaidPluginsAtOnce"
+              >
+                <h2>{{ translate('Marketplace_InstallAllPurchasedPlugins') }}</h2>
+                <p>
+                  {{ translate('Marketplace_InstallThesePlugins') }}
+                  <br /><br />
+                </p>
+                <ul>
+                  <li v-for="pluginName in paidPluginsToInstallAtOnce" :key="pluginName">
+                    {{ pluginName }}
+                  </li>
+                </ul>
+                <p>
+                  <input
+                    role="install"
+                    type="button"
+                    :href="installAllPaidPluginsLink"
```

Comment:
This needs to be a `data-href` attribute in order to work:
See https://github.com/matomo-org/matomo/blob/ee41a13e975d5ffeae278a78b786c8326ead694b/plugins/Morpheus/javascripts/piwikHelper.js#L412-L416

Reply:
:+1: fixed